hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -1454,7 +1455,7 @@ public CacheRecord merge(SplitBrainMergeEntryView<Data, Data> mergingEntry, Spli
         final long now = Clock.currentTimeMillis();
         final long start = isStatisticsEnabled() ? System.nanoTime() : 0;
 
-        mergePolicy.setSerializationService(nodeEngine.getSerializationService());
+        SplitBrainUtil.injectDependencies(mergePolicy, nodeEngine);
```

Comment:
Can't use static imports here, since the class already has a method with this name.